measureradialentropy.py

```diff
@@ -77,7 +78,11 @@ def run(self, workspace):
 
         indexes = objects.indices
         #Calculate the center of the objects- I'm guessing there's a better way to do this but this was already here
-        centers, radius = minimum_enclosing_circle(labels, indexes)
+        #centers, radius = minimum_enclosing_circle(labels, indexes)
```

Comment:
You can drop the import at the top of the file and remove this commented code. The rationale behind using regionprops over minimum_enclosing_circle is better captured in a commit message.